HypVINN/utils/img_processing_utils.py

```diff
@@ -74,10 +118,37 @@ def save_logits(
         logits: npt.NDArray[float],
         orig_path: Path,
         ras_affine: npt.NDArray[float],
-        ras_header,
+        ras_header: nib.nifti1.Nifti1Header,
```

Comment:
I think this is incorrect and it should have more types that just the nifty1header, at a minimum nifty2 and MGH Header should also be fine.